### ISSUE TITLE
Rewrap shouldn't add a new line if the characters that 'go over' are space characters

### ISSUE THREAD

**Issue by AncientSwordRage** [stkb/Rewrap#305]
I noticed this behaviour:

```javascript
// pretend this line is at maximum allowed length.
// and there a line here
```

Running rewrap there makes not changes.

But before I noticed it, I added a double-space to the end (not realising it was at max)

(nb, I'm using `·` to show a space character)

before:

```javascript
//·pretend this line is at maximum allowed length.··
//·and there a line here
```
After:

```javascript
//·pretend this line is at maximum allowed length.
//·
//·and there a line here
```

And also:

```javascript
//·pretend this line is at maximum allowed length.········
//·
//·and there a line here
```
After

```javascript
//·pretend this line is at maximum allowed length.
//·
//
//·and there a line here
```

I don't know if it should preserve any number of trailing spaces, truncate it to two (if there were more than to) or continue adding new blank lines, but I think it's confusing that it does this

**Comment by stkb** [comment]
Hi, thanks yeah this was wrong behavior and I believe I fixed this in 5fc54c1. The fix will be in the next version which I'm working on releasing right now. ([more info](https://stkb.github.io/Rewrap/specs/features/spaces/)

**Comment by stkb** [comment]
Should be fixed in v1.16.0. Let me know if not.
